concrete/single_pages/dashboard/reports/security.php

```diff
@@ -4,62 +4,84 @@
  */
 use Concrete\Core\Validator\ErrorLevel;
 
-foreach ($result as $validatorName => $resultSet) {
-    $severities = array_map(function(\Concrete\Core\Validator\Site\SiteError $error) {
-        return $error->getSeverity();
-    }, $resultSet);
+if (!count($result)) {
+    ?>
+     <div class="col-sm-12">
+        <div class="alert alert-success text-center">
+            <?= t('No issues found!') ?>
```

Comment:
Originally we listed the tests with green success outlines, are you sure we want to have this instead? I figured it would make people feel good to have the page filled with green checks.